soc/xtensa/intel_adsp/Kconfig

```diff
@@ -21,6 +21,22 @@ config HP_SRAM_RESERVE
 	  addresses here.  The SOF application also makes direct use
 	  of this region, so be very careful changing this value.
 
+config ADSP_WIN0_OFFSET
+	int "Byte offset of host window zero"
+	default 16384
+	help
+	  Offset of the first host shared memory window (used for the
+	  "firmware status register" at boot) within the reserved
+	  region at the start of HP-SRAM.
+
```

Comment:
is long term thinking to partition the windows and permissions in DT ? Seem like a good place ?